Code/GraphMol/StereoGroup.h

```diff
@@ -0,0 +1,48 @@
+//
+//  Copyright (C) 2003-2015 Greg Landrum and Rational Discovery LLC
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+/*! \file StereoGroup.h
+
+  \brief Defines the class StereoGroup which stores relationships between
+  the absolute configurations of atoms within a structure.
+
+*/
+
+#include <RDGeneral/export.h>
+#ifndef __RD_StereoGroup_H__
```

Comment:
The include guards shouldn't start with a double underscore. Lots of existing RDKit code does this still, but I fix it when I run across it (and think to do so).
Please use `RD_STEREO_GROUP_092018` or something like that

Reply:
Have you considered using `#pragma once`?

Reply:
As long as that works across modern compilers (and I think it does) I’m fine with it.